ebsco_adapter/ebsco_adapter/tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+envlist = py3
+skipsdist = True
+
+[testenv]
+basepython = python3
+deps =
+    -e ../../common/catalogue_python_fixtures
+    -r{toxinidir}/src/test_requirements.txt
+commands =
+    py.test --cov={toxinidir}/src {posargs}
```

Comment:
This is fine, but it does produce a coverage report that includes the test file, which I would expect to always be 100%
```
Name                      Stmts   Miss  Cover
---------------------------------------------
src/main.py                  15      8    47%
src/test_placeholder.py       4      0   100%
---------------------------------------------
TOTAL                        19      8    58%
```

Personally, I'm not a fan of simple numeric coverage reports like this unless we are using it to ensure that our coverage doesn't fall between commits (and here is where skewing the result by recording coverage in the test files can be bad).  The real value is seeing exactly where there's a gap in coverage, rather than just knowing that half the lines are not exercised by the test suite.

Not a blocker at all, just something that. might trip us up in the future.

Reply:
This is what the others python lambdas do copied thoughtlessly here, I like a coverage report as long as it's actually visible somewhere.